docs/project/changelog.md

```diff
@@ -46,6 +46,10 @@ substitutions:
   {any}`JsProxy.to_memoryview` to allow for conversion of `TypedArray` to
   standard Python types without unneeded copies. {pr}`1864`
 
+- {{Enhancement}} Added {any}`JsProxy.to_file` and {any}`JsProxy.from_file` to
+  allow reading and writing Javascript buffers to files without unneeded copies.
```

Comment:
Maybe,

```suggestion
  allow reading and writing Javascript buffers to files as a byte stream without unneeded copies.
```
as otherwise the question is what format we use for serialization.